data/json/recipes/recipe_deconstruction.json

```diff
@@ -7708,4 +7708,44 @@
     "qualities": [ { "id": "HAMMER", "level": 2 }, { "id": "SAW_M", "level": 2 } ],
     "components": [ [ [ "steel_lump", 7 ], [ "steel_chunk", 2 ] ] ]
   }
+  {
+    "result": "fish",
+    "type": "uncraft",
+    "activity_level": "NO_EXERCISE",
+    "time": "20 s",
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [ [ [ "fish_scrap", 10 ] ] ]
+  },
+  {
+    "result": "meat",
+    "type": "uncraft",
+    "activity_level": "NO_EXERCISE",
+    "time": "20 s",
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [ [ [ "meat_scrap", 10 ] ] ]
+  },
+  {
+    "result": "human_flesh",
+    "type": "uncraft",
+    "activity_level": "NO_EXERCISE",
+    "time": "20 s",
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [ [ [ "human_meat_scrap", 10 ] ] ]
+  },
+  {
+    "result": "demihuman_flesh",
+    "type": "uncraft",
+    "activity_level": "NO_EXERCISE",
+    "time": "20 s",
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [ [ [ "demihuman_meat_scrap", 10 ] ] ]
+  },
+  {
+    "result": "mutant_meat",
+    "type": "uncraft",
+    "activity_level": "NO_EXERCISE",
+    "time": "20 s",
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [ [ [ "mutant_meat_scrap", 10 ] ] ]
+  },
```

Comment:
```suggestion
  }
```

Reply:
Good catch.  This also reminded me that the previous last entry didn't have a comma, so I added that in as well.